test/OpenCensus.Exporter.ApplicationInsights.Tests/Implementation/TraceExporterHandlerTests.cs

```diff
@@ -0,0 +1,1949 @@
+            var url = new Uri("https://host:123/path?query");
+            name = "HttpOut";
+            kind = SpanKind.Client;
+            attributes = Attributes.Create(new Dictionary<string, IAttributeValue>()
+                {
+                    { "http.url", AttributeValue.StringAttributeValue(url.ToString()) },
+                    { "http.method", AttributeValue.StringAttributeValue("POST") },
+                    { "http.status_code", AttributeValue.LongAttributeValue(200) },
+                }, 0);
+            var span = SpanData.Create(context, parentSpanId, hasRemoteParent, name, startTimestamp, attributes, annotations, messageOrNetworkEvents, links, childSpanCount, status, kind, endTimestamp);
+
+            var sentItems = this.ConvertSpan(span);
+
+            var dependency = sentItems.OfType<DependencyTelemetry>().Single();
+            Assert.Equal(url.ToString(), dependency.Data);
+            Assert.Equal("POST /path", dependency.Name);
+            Assert.Equal("200", dependency.ResultCode);
+            Assert.Equal("host", dependency.Target);
+            Assert.Equal("Http", dependency.Type);
+        }
+
+        [Fact]
+        public void OpenCensusTelemetryConverterTests_TracksHttpDependencyWithRelativeUrl()
+        {
+            this.GetDefaults(out var now, out var context, out var parentSpanId, out var hasRemoteParent, out var name, out var startTimestamp, out var attributes, out var annotations, out var messageOrNetworkEvents, out var links, out var childSpanCount, out var status, out var kind, out var endTimestamp);
+            var url = new Uri("https://host:123/path?query");
+            name = "HttpOut";
+            kind = SpanKind.Client;
+            attributes = Attributes.Create(new Dictionary<string, IAttributeValue>()
+                {
+                    { "http.url", AttributeValue.StringAttributeValue(url.LocalPath) },
+                    { "http.method", AttributeValue.StringAttributeValue("POST") },
+                    { "http.status_code", AttributeValue.LongAttributeValue(200) },
+                }, 0);
+            var span = SpanData.Create(context, parentSpanId, hasRemoteParent, name, startTimestamp, attributes, annotations, messageOrNetworkEvents, links, childSpanCount, status, kind, endTimestamp);
+
+            var sentItems = this.ConvertSpan(span);
+
+            var dependency = sentItems.OfType<DependencyTelemetry>().Single();
+            Assert.Equal(url.LocalPath, dependency.Data);
+            Assert.Equal("POST /path", dependency.Name);
+            Assert.Equal("200", dependency.ResultCode);
+            Assert.Null(dependency.Target);
+            Assert.Equal("Http", dependency.Type);
+        }
+
+        /*
+        [Fact]
+        public void OpenCensusTelemetryConverterTests_TracksHttpDependencyWithUrlIgnoresHostPortPath()
```

Comment:
why so many tests are commented out?

Reply:
Fixed most of them. Tests with the node are still commented out - we need some Node API